### PR TITLE
Readme.md

### PR DESCRIPTION
**Pull request by MrRaccoonUA**
# What does this pull request do?
Complete README.md

### CODE REVIEW

**Comment by MrRaccoonUA** [comment]
@aartiukh please, check only `067f4c5100f8d0b3a767eeccf0f08b1e5295dff7` this commit

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+
+## Usage
+
+**UPC (Universal Product Code)** - American standard barcode for tracking goods in stores.
+
+This script works on the API of the [upcitemdb.com](https://www.upcitemdb.com/). The "trial" version is used in the script, which means that you can make a request for 100 UPC per day. 
+
+**Steps to run a script**
+
+- Open file `main.py`
```
Comment (aartiukh) [inline README.md:18]: Why do we need this step?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+- Install `requirements.txt` with command `pip install -r requirements.txt`
+- Run script
```
Comment (aartiukh) [inline README.md:20]: It would be better to provide a command line as an example how to run your script

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+
+**Manual setting**
+
+- There is a timeout setting in the script `API_COOL_DOWN_TIMEOUT_SECS`. By default, there is a delay of 10 seconds, because the trial is used.
+- To change the plan, in the `API_URL_GET` line, change `/trial/` to `/v1/`
+- You can also change the minimum allowed image resolution. The default is 600px by 600px. `WIDTH_DIMENSION` and `HEIGHT_DIMENSION` respectively
```
Comment (aartiukh) [inline README.md:26]: these configs should be passed through command line, if empty - use default ones

File: README.md (hunk 2 above)
Comment (MrRaccoonUA) [reply README.md:20]: Fixed

File: README.md (hunk 1 above)
Comment (MrRaccoonUA) [reply README.md:18]: Fixed

File: README.md (hunk 3 above)
Comment (MrRaccoonUA) [reply README.md:26]: I will consider it

Comment (aartiukh) [reply README.md:26]: Do me a favor 😉

**Review by aartiukh — APPROVED** [review]

**Pull request merged**